src/overlays/actors/ovl_En_Peehat/z_en_peehat.c

```diff
@@ -927,7 +924,7 @@ void EnPeehat_Update(Actor* thisx, GlobalContext* globalCtx) {
     Player* player = PLAYER;
 
     // If Adult Peahat
-    if (thisx->params <= 0) {
+    if (thisx->params <= 0) { // thisx is required to match
```

Comment:
i dont think its worth commenting personally, we assume when its used its needed. and it happens somewhat often, so commenting would get redundant

Reply:
this might have been missed, i dont think we should add this in